resources/asciidoctor/spec/elastic_include_tagged_spec.rb

```diff
@@ -13,148 +13,174 @@
     Asciidoctor::Extensions.unregister_all
   end
 
-  it "includes a tag" do
-    actual = convert <<~ASCIIDOC
+  INCLUDE_FILE = 'resources/elastic_include_tagged/Example.java'
```

Comment:
It seem strange to define a constant in a spec file. Maybe you can pass in a `include_file` variable to the method and define in a `let` block so it can also be used [here](https://github.com/elastic/docs/pull/776/files#diff-66f24cf664d01c71e63a3626975d6d7cR138)